src/redux/country/countries.js

```diff
@@ -0,0 +1,76 @@
+import axios from 'axios';
+
```

Comment:
- [ ] You should create test cases for pure functions too. 

From Project Specification :arrow_down: 
>Testing requirements
>
>    Create unit tests for pure functions in your Redux code.